### DISCUSSION TITLE
Welcome to AppleIGB Discussions!

### DISCUSSION THREAD

**Discussion by donatengit** [donatengit/AppleIGB#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Leborgne23** [comment]
Hi ! First of all thanks for undertaking this task. In my case the ethernet connexion now survives sleep mode but internet connectivity breaks down after a few minutes, sometimes a few hours. Manually switching duplex / speed in sysprefs brings it back to a working state. Looking forward to seing the improvements on that matter. Thanks again./

**Reply by donatengit** [reply]
Hi @Leborgne23 

Thanks for checking this out. Could you please provide a bit more info for diagnosis: you NIC, macOS version, what is your connection peer (router, internet provider directly, etc), output of _ifconfig -v_ for your _enX_ interface (you could remove _ether, inet, inet6_ data if that matters for privacy)?

**Comment by donatengit** [comment]
This fork is being deprecated, see [README](https://github.com/donatengit/AppleIGB) for better option of the driver.
